salt/metalk8s/roles/etcd/prepared.sls

```diff
@@ -0,0 +1,3 @@
+include:
```

Comment:
This is not a 'role', not sure it belongs here.

Reply:
Done